packages/webapp/src/components/Task/PureTaskDetails/index.jsx

```diff
@@ -138,11 +138,7 @@ export default function PureTaskDetails({
         >
           {isHarvest
             ? t('ADD_TASK.HOW_MUCH_IS_HARVESTED')
-            : t('ADD_TASK.TELL_US_ABOUT_YOUR_TASK_TYPE_ONE') +
-              ' ' +
-              (isCustomType ? taskName.toLowerCase() : t(`task:${taskType}_LOWER`)) +
-              ' ' +
-              t('ADD_TASK.TASK')}
+            : t('ADD_TASK.TELL_US_ABOUT_YOUR_TASK_TYPE_ONE')}
```

Comment:
Made this string more generic because this structure really only makes sense in English

Reply:
The ideal translation should be `"Tell us about the {{taskType}}"` (translations should be written using `{{taskType}}`), but since we don't have the time, I will approve it!

Reply:
Yeah that was what I originally was going to go for, but in French it gets complicated because for some strings you need a contraction, i.e. "Décrivez cette tache d’irrigation” for irrigation tasks, whereas for others it's just "Décrivez cette tache de nettoyage". In the end we were talking about it with Loïc and we decided to go for the more generic copy which seems to adapt better to different languages